contributors/devel/sig-testing/e2e-tests.md

```diff
@@ -25,6 +25,7 @@
   - [Kinds of tests](#kinds-of-tests)
     - [Viper configuration and hierarchichal test parameters.](#viper-configuration-and-hierarchichal-test-parameters)
     - [Conformance tests](#conformance-tests)
+	- [Validation suites](#validation-suites)
```

Comment:
The structure here implies that you consider a Validation Suite to be a kind of Conformance Test?  If so, I think that should be made more clear in the body of the document, because currently it's not.

Reply:
Either that, or change the structure to be more consistent with the statement in the doc that "a Validation Suite is a subset of e2e tests".

Reply:
Structure is wrong.  They are peers. Validation is not a kind of conformance test.  I will fix

Reply:
I had a hidden tab in there :-). It's been removed